deeplearning4j-nlp-parent/deeplearning4j-nlp-japanese/src/main/java/org/deeplearning4j/text/tokenization/tokenizerfactory/KuromojiBinFilesFetcher.java

```diff
@@ -0,0 +1,70 @@
+package org.deeplearning4j.text.tokenization.tokenizerfactory;
+
+import org.apache.commons.io.FileUtils;
+import org.deeplearning4j.util.ArchiveUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by kepricon on 16. 11. 24.
+ */
+public class KuromojiBinFilesFetcher {
+    protected static final Logger log = LoggerFactory.getLogger(KuromojiBinFilesFetcher.class);
+    protected static final String TEMP_ROOT = System.getProperty("java.io.tmpdir");
```

Comment:
Don't use tmp directory. We want these models to be saved. Put it under the user's home directory in a hidden folder.